tensorrt_llm/_torch/pyexecutor/py_executor_creator.py

```diff
@@ -329,14 +334,16 @@ def create_py_executor(
             spec_config, model_engine, draft_model_engine)
         if spec_resource_manager is not None:
             resources["spec_resource_manager"] = spec_resource_manager
+        if spec_config.spec_dec_mode.is_ngram():
+            drafter.pool_manager = spec_resource_manager
```

Comment:
I suspect that other `Drafter`s will need resource managers in the future. Can we instead create the drafter like this:
```python
class Drafter:
    def __init__(self, ..., resource_manager: Optional[ResourceManager] = None) 
```